src/components/reports/IndividualScoreReportTask.vue

```diff
@@ -24,6 +24,7 @@
               </div>
             </div>
             <PvKnob
+              disabled
```

Comment:
Shouldn't this be `readOnly`? See https://primevue.org/knob/#api.knob.props.readonly